### PR TITLE
Finalize text message and Actions slot API

### PR DESCRIPTION
**Pull request by hamen**
image

### CODE REVIEW

File: ui/src/main/kotlin/org/jetbrains/jewel/ui/component/Banner.kt [hunk 1]
```diff
@@ -78,18 +80,26 @@ private fun BannerImpl(
     actions: (@Composable RowScope.() -> Unit)?,
     modifier: Modifier = Modifier,
 ) {
-    Column(modifier = modifier) {
+    Column(
+        modifier = modifier.height(40.dp).background(style.colors.background),
+        verticalArrangement = Arrangement.Center,
```
Comment (rock3r) [inline ui/src/main/kotlin/org/jetbrains/jewel/ui/component/Banner.kt:85]: I don't think this does what you think it does :D And I'd argue this is wrong to have

File: ui/src/main/kotlin/org/jetbrains/jewel/ui/component/Banner.kt [hunk 2]
```diff
@@ -78,18 +80,26 @@ private fun BannerImpl(
...
-            Spacer(modifier = Modifier.weight(1f))
+            Text(
+                text = text,
+                style = textStyle,
+                maxLines = 1,
+                overflow = TextOverflow.Ellipsis,
+                modifier = Modifier.weight(1f),
+            )
             if (actions != null) {
                 Row(
```
Comment (rock3r) [inline ui/src/main/kotlin/org/jetbrains/jewel/ui/component/Banner.kt:104]: I think we need to add a Spacer before this Row, so there is room between the text and the actions

Comment (rock3r) [reply ui/src/main/kotlin/org/jetbrains/jewel/ui/component/Banner.kt:104]: Inside the if, of course :)

Comment (hamen) [reply ui/src/main/kotlin/org/jetbrains/jewel/ui/component/Banner.kt:104]: No spacer
image

Spacer
image

I'm adding the Spacer

Comment (hamen) [reply ui/src/main/kotlin/org/jetbrains/jewel/ui/component/Banner.kt:105]: Done in https://github.com/JetBrains/jewel/pull/698/commits/0e16598c6b0377735fd4b174998edbf623e1f112

File: ui/src/main/kotlin/org/jetbrains/jewel/ui/component/Banner.kt (hunk 1 above)
Comment (hamen) [reply ui/src/main/kotlin/org/jetbrains/jewel/ui/component/Banner.kt]: Done in https://github.com/JetBrains/jewel/pull/698/commits/0ae8318434bbed30eb918f713ba772d2c2c80d9d

**Review by rock3r — APPROVED** [review]

**Pull request merged**